cli/commands.go

```diff
@@ -24,6 +24,37 @@ import (
 )
 
 const defaultRegistry = "docker.io"
+const clusteNameMaxSize int = 35
```

Comment:
MIssing an `r` here? :thinking:

Reply:
Thanks for catching it. I will push a fix soon.